### DISCUSSION TITLE
[Q/A][Java SDK (Component)] Dubbo  application mapping relationship when will it be corrected?

### DISCUSSION THREAD

**Discussion by iwangjie** [apache/dubbo#14491] [Question]
### Pre-check

- [X] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

For example, I have a service whose service name is incorrectly filled in, causing Dubbo/mapping/XXX. exe. The value of Aservice appears as XXX1 and XXX2, and then Aservice corrects the package name in time for the XXX2 service, but/Dubbo/mapping/XXX. When will AService's XXX2 be cleaned up?

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
Currently, we cannot remove it unless we have a good plan
